frame/support/src/inherent.rs

```diff
@@ -64,21 +69,30 @@ pub trait ProvideInherent {
 	/// included in the block by its author. Whereas the second parameter represents the inherent
 	/// data that the verifying node calculates.
 	///
-	/// NOTE: A block can contains multiple inherent.
+	/// This is intended to allow for checks that cannot be done within the runtime such as, e.g.,
+	/// the timestamp.
+	///
+	/// NOTE: A block can contain multiple inherents.
```

Comment:
How should this note help the user?

Reply:
I assume just a hint that `check_inherent` can be called more than once for the same (kind of) inherent? I just left it as is, happy to write something that makes more sense/is easier to understand.

Reply:
Yes this was my intention, a hint that this will be called on each inherent with a call of the type of the call of the pallet.

But thinking again it is not really useful. The is `check_inherent` and it does check for all inherent with a call of the type of the call of the pallet.